src/modules/addComment.js

```diff
@@ -0,0 +1,23 @@
+import { postComment } from './retrieveshow.js';
+import baseURL from './api.js';
+
+class AddComment {
```

Comment:
- [ ] Kindly fix the comments form, I have failed to add a comment. I have observed the same issue even with the deployed app.

Reply:
Hi @elisha2kyakpo1, thank you very much for your correct observation.
We did not understand the reason at first because the comment functionality works on our laptops. So we run 'npm run build' and update the files on Github.
I tested it and now it is working. Please try again.